scan/main.py

```diff
@@ -74,22 +94,22 @@ def _run_crew(crew):
 
 
 def main() -> int:
-    print("## Welcome to the SCAN")
+    print("## Welcome to the SCAN System")
     print("---------------------------------------------------------------")
     continue_analysis = True
     while continue_analysis:
-        topic = input("Please enter the topic you want to analyze: ")
+        topic = input("Please enter the topic you need help with: ")
         print("You entered:", topic)
         custom_crew = CustomCrew(topic)
         result = custom_crew.run()
         print("\n\n########################")
-        print("## Crew AI Operation Result:")
+        print("## SCAN AI Operation Result:")
         print("########################\n")
         print(result)
-        response = input("Do you want to analyze another topic? (yes/no): ")
-        if response.lower() != "yes":
-            continue_analysis = False
 
+        continue_analysis = False
```

Comment:
This will always only run once now so no need for the `while` loop any more.

```suggestion
    topic = input("Please enter the topic you need help with: ")
    print("You entered:", topic)
    custom_crew = CustomCrew(topic)
    result = custom_crew.run()
    print("\n\n########################")
    print("## SCAN AI Operation Result:")
    print("########################\n")
    print(result)
```